roles/network_plugin/calico/tasks/install.yml

```diff
@@ -207,6 +207,7 @@
           "name": "default",
       },
       "spec": {
+          "listenPort": {{ calico_bgp_listen_port|default(179) }},
```

Comment:
you don't need to put `default(179)` here

Reply:
yeah it's a leftover, thanks. Removed :)